controllers/music.js

```diff
@@ -0,0 +1,31 @@
+const Music = require("../models/music");
+const { StatusCodes } = require("http-status-codes");
+const { NotFoundError, BadRequestError } = require("../errors");
+const getAllMusics = async (req, res) => {
+  res.send("get all music");
+};
+const getMusic = async (req, res) => {
+  res.send("get one of the music");
+};
+const createMusic = async (req, res) => {
+  // res.send("create a music");
+  //res.json(req.user);
+  //res.json(req.body);
+  req.body.createdBy = req.user.userId;
+  const music = await Music.create(req.body);
```

Comment:
Make sure to keep the same format here of `{...req.body}` to ensure you're passing an object of the correct structure